### ISSUE TITLE
Why is the slip reward about the angular velocity instead of the linear velocity for feet?

### ISSUE THREAD

**Issue by m4k1se** [AgibotTech/agibot_x1_train#10]
```python
   def _reward_foot_slip(self): 
        """
        Calculates the reward for minimizing foot slip. The reward is based on the contact forces
        and the speed of the feet. A contact threshold is used to determine if the foot is in contact
        with the ground. The speed of the foot is calculated and scaled by the contact conditions.
        """
        contact = self.contact_forces[:, self.feet_indices, 2] > 5.
        foot_speed_norm = torch.norm(self.rigid_state[:, self.feet_indices, 10:12], dim=2)
        rew = torch.sqrt(foot_speed_norm)
        rew *= contact
        return torch.sum(rew, dim=1)
````

index [10:12] means the wx, wy of rigid，but common sense would suggest that foot sliding should be more related to linear velocity, so why is it set to angular velocity here?
